### ISSUE TITLE
Autotype not working

### ISSUE THREAD

**Issue by ArkitectNaut** [MacPass/MacPass#1216]
**Describe the bug**
I have followed all the necessary steps from the Wiki to implement the Autotype feature but for the life of me can't get it to work. I have given it the accessibility access, given the name for the window, provided the website's URL but it still doesn't work. 

**To Reproduce**
1. Create entry
2. Fill in all needed info
3. Go to website's login page
4. Go back to MacPass and use Autotype

**Expected behavior**

Fill in the username and password

**Desktop (please complete the following information):**
 - macOS: 12.2.1
 - Version 0.8.1

**Additional context**
I don't know if I am missing a step but I've honestly spent 30min trying to get it working. If it isn't a bug and it's just me missing some crucial step, I think it would be a worthwhile addition to the Wiki having a short video showing the process.

_PS - any news on importing CSV files?_

**Comment by mstarke** [comment]
CSV import is still on hold but I did get quite far so it might not take that much longer.

**Comment by mstarke** [comment]
To clarify, you are trying to use Autotype. Autotype will re-activate the previous active window before MacPass got focus and type the credentials according to the autotype sequence (default is: ```{USERNAME}{TAB}{PASSWORD}{ENTER}```

Autotype does not require any information it will simply use the entry you have selected.

In your case:

I did add some **bold** steps to make sure you aren't missing any necessary steps. The *italic* ones are what should happen automatically

1. Create entry
2. Fill in all needed info
3. Go to website's login page
4. **Place the cursor in the user-input-form**
5. Go back to MacPass
6. **Select the correct Entry**
7. Use Autotype
8. *The Website window should get the front-most window again*
9. *MacPass should type in the username and password in the form*

**Comment by ArkitectNaut** [comment]
I've been using Netflix to test. In terms of info, I have provided:
- Title: Netflix
- Username
- Password
- Url: https: https://www.netflix.com
- Window Title: www.netflix.com (I have the '_use host as window title_' option selected)
- Window Sequence: {USERNAME}{TAB}{PASSWORD}{ENTER}
- Enable Autotype: check

I than save changes, go to Netflix's login page, click the '_email or phone number_' field, switch to MacPass, select the Netflix password entry, click the Autotype button (have also tried cmdT) and nothing happens.

**Comment by mstarke** [comment]
If you open the `Window -> Autotype Doctor`, are both status bubbles green?
<img width="568" alt="Screenshot 2022-02-23 at 12 29 13" src="https://user-images.githubusercontent.com/58428/155311041-5139ff8a-9ac7-4664-867e-a54ca2d24cfb.png">

**Comment by ArkitectNaut** [comment]
Mate that just solved it! I hadn't provided permission to record the screen. The explanation text plus the dividing line between the two options seemed to suggest it was only needed for Global Autotype.

Thank you for your help and instant responses.

**Comment by mstarke** [comment]
Awesome. And you are right, the explanation is misleading. I will have to change this since it's not intuitive. If you select an entry, there should be no reason to read the window title. Only Global Autotype should require it *but* even if you select an entry, you might have different window-association defined and MacPass still needs to query for the window title to the right match.

I'm closing this since it's solved for you!

**Comment by mstarke** [comment]
What wonders me it that MacPass should have complained about not having all permission to run. Do you have Notifications enabled for MacPass?

**Comment by ArkitectNaut** [comment]
> Do you have Notifications enabled for MacPass?

I did not but sounds like it would have been helpful.
